deployment/start_and_deploy_to_server.sh

```diff
@@ -31,12 +31,12 @@ persistent_keycloak_backup_dir=/home/ubuntu/persistent_keycloak_backup
 keycloak_user_dir=$location/dataland-keycloak/users
 
 # shut down currently running dataland application and purge files on server
-ssh ubuntu@"$target_server_url" "cd \"$location\" && sudo docker compose down" || true
+ssh ubuntu@"$target_server_url" "(cd \"$location\" && sudo docker compose down) || true"
```

Comment:
Macht das einen Unterschied? Sollte das ssh statement nicht mit dem gleichen return code zurückkommen wie der ausgeführte Befehl?

Reply:
@fsoffel Kannst du das beantworten?

Reply:
Der Gedanke war, dass ja auch eine SSH Verbindung nicht established werden kann. Und in dem Fall darf es gerne einen Fehler geben.